overlord/devicestate/firstboot_test.go

```diff
@@ -1065,7 +1091,7 @@ snaps:
 }
 
 func (s *FirstBootTestSuite) TestImportAssertionsFromSeedClassicModelMismatch(c *C) {
-	release.OnClassic = true
+	release.MockOnClassic(true)
```

Comment:
Do you need to restore this?

Reply:
The firstboot suite does "	s.restoreOnClassic = release.MockOnClassic(false)" in SetUpTest and does the restore in "TearDownTest" - if you prefer this to have a restore just for easier reading I'm happy to add it.

Reply:
that's why it was written the other way btw

Reply:
I know, I can revert its just that when I searched for classic tests I did a search for MockOnClassic() and was confused. I can revert this if you prefer the old style.

Reply:
I'm fine either way myself, I have sympathy for people expecting the restore in this style tough